.gitignore

```diff
@@ -14,9 +14,12 @@ gpg.*
 windows/**/wix.dynamic.json
 windows/**/wix
 windows/config.yaml
+windows/logging.yaml
 windows/LICENSE
 windows/plugins
 opentelemetry-java-contrib-jmx-metrics.jar
 VERSION.txt
 release_deps
 tmp
+go.work
+go.work.sum
```

Comment:
We shouldn't have these in the .gitignore. We don't support this go1.18 feature but we might want to in the future.